キャッシュクリア機能追加

tacko-o opened the pull request:
#717
一時ファイルを削除するボタンを「設定>全般設定」の一番下に追加
キャッシュクリアに関するローカライゼーション用の文言の追加

【キャッシュクリア時のイメージ】
クリア前
![before](https://github.com/user-attachments/assets/0dbeafde-4619-460d-8b84-d99994efdf2c)

クリア後 (8~30MB程度 一時ファイル以外のデータが残る模様)
![after](https://github.com/user-attachments/assets/9b60b27b-ea4f-4622-99ef-a76c55a2c86e)

tacko-o committed:
キャッシュクリア機能追加

全般設定にキャッシュクリア機能を追加
キャッシュクリアに関するl18n関連の文言追加

tacko-o committed:
コメント修正

コメント脱字修正

tacko-o committed:
不要なimport削除

不要なimport削除

4ster1sk commented:
Androidの場合、キャッシュファイル数が多いとサイズ計算にかなり時間がかかってしまって、全般設定画面がしばらくフリーズしてしまうので非同期で計算したほうがいいかも。

poppingmoon commented:
`Directory.list(recursive: true)` や `Directory.delete(recursive: true)` を使うともう少しシンプルになると思います

画像のキャッシュを消すだけなら `DefaultCacheManager().emptyCache()` を使ってもいいかもしれません

tacko-o commented:
`Directory.list(recursive: true)` `Directory.delete(recursive: true)`だとシンプルかつ非同期にできそうなので変えてみます
`DefaultCacheManager().emptyCache()`はドキュメントには「キャッシュを全部消す」と書いてはあるものの、実際にやってみたところキャッシュが保存されているディレクトリとその配下のファイルにはなんの変化も起こらず……
しばらく待ったりアプリを再起動したりしてみてもキャッシュが消える様子はなかったので、環境によっては(うちがmacだから?)うまく動かないのかもしれません
libCachedImageData/配下のファイルがキャッシュの大部分だと思うので、今回はこのディレクトリを直接削除するようにしてみます

tacko-o committed:
キャッシュの取得削除を非同期にするように

キャッシュされたファイルが多いとキャッシュサイズ取得中にUIが固まってしまうため、キャッシュサイズ取得を非同期で行うように変更
同じ理由でキャッシュクリア時も非同期で行うように変更
getTemporaryDirectory()はLinuxだと/tmp/が返却されてしまうので、getTemporaryDirectory()配下のファイルをすべて削除するのではなくlibCachedImageData/のみを削除するように変更
